testing/trino-product-tests/src/main/resources/sql-tests/testcases/horology_functions/checkHorologyFunctionsRegistered.result

```diff
@@ -66,7 +66,6 @@
  quarter | bigint | timestamp(p) with time zone | scalar | true | Quarter of the year of the given timestamp |
  timezone_hour | bigint | timestamp(p) with time zone | scalar | true | Time zone hour of the given timestamp |
  timezone_minute | bigint | timestamp(p) with time zone | scalar | true | Time zone minute of the given timestamp |
- to_unixtime | double | timestamp(p) | scalar | true | |
```

Comment:
Shows up in `Return timestamp with time zone from from_unixtime` but belongs to prev commit